src/Page.test.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import Page from './Page';
+
+describe('Page', () => {
+  const handleChangeTitle = jest.fn();
+  const handleClickAddTask = jest.fn();
+  const handleClickDeleteTask = jest.fn();
+  const taskTitle = '';
+
+  function renderPage(
+    {
+      tasks,
+    },
+  ) {
```

Comment:
인자가 하나밖에 없을땐 줄바꿈을 굳이 안해줘도 될 것 같아요~ 그리고 객체로 받을 필요도 없을 것 같구요.
```suggestion
  function renderPage(tasks) {
```